src/remodal.js

```diff
@@ -431,6 +422,22 @@
   }
 
   /**
+   * Generates a string separated by dashes and prefixed with NAMESPACE
+   * @private
+   * @param str, str, ...
```

Comment:
http://usejsdoc.org/tags-param.html#multiple-types-and-repeatable-parameters
https://developers.google.com/closure/compiler/docs/js-for-compiler?csw=1
http://stackoverflow.com/a/7954617

They are different everywhere. But i prefer to write a type with a capitalized first letter.